### PR TITLE
NC class for bio-weapon NPCs

### PR DESCRIPTION
**Pull request by chaosvolt**
* Made bio-weapon NC class a thing that works. Super soldier uniform, focus on FN and Rivtech rifles.
* Gave the bio-weaps a "generic bio-weapon" marker trait, because Noct's suggested traits and loadout doesn't match any of the bio-weapon professions.
* Trigger and Blade no longer say they lost their badges. 3:

### CODE REVIEW

File: Mutations/c_bio_mutation.json [hunk 1]
```diff
@@ -41,6 +41,16 @@
   },
   {
     "type": "mutation",
+    "id": "BIO_WEAPON_GENERIC",
+    "name": "Bio-Weapon",
+    "points": 1,
+    "description": "This DNA marker serves as a reminder of what you are - a Bio-Weapon.  This is for NPCs, if you have this it's a bug to pester Noctifer about!",
```
Comment (Noctifer-de-Mortem) [inline Mutations/c_bio_mutation.json]: That description lol!

Comment (chaosvolt) [inline Mutations/c_bio_mutation.json]: When in doubt, annoy Noct. o3o

**Comment by Noctifer-de-Mortem** [comment]
Gonna give a proper test soon and might do a couple additions if you don't mind.

**Comment by chaosvolt** [comment]
Ah, sure. Anything you had in mind? I still intend to eventually fiddle with the lab stuff to expand them.

**Comment by Noctifer-de-Mortem** [comment]
Just a couple of other mutations that are not 100 percent guaranteed to have.

**Comment by chaosvolt** [comment]
Ah, makes sense.

**Comment by chaosvolt** [comment]
Ah, neat. <3

**Pull request merged**